Content for tabs (#907)
novuhq/docs: content/docs/platform/inbox/configuration/tabs.mdx

icon: 'Folders'
---

> In the Inbox components, you can use Tabs to organize in-app notifications into separate sections, creating a cleaner, more organized experience for your subscribers. Each tab displays a subset of in-app notifications based on defined rules, making it easier to surface important notifications.

Reviewer: ```suggestion
In the Inbox components, you can use Tabs to organize **Inbox** notifications into separate sections, creating a cleaner, more organized experience for your subscribers. Each tab displays a subset of in-app notifications based on defined rules, making it easier to surface important notifications.
```

> Tabs in the Inbox component are used to organize in-app notifications into separate sections, creating a cleaner, more organized experience for your subscribers. Each tab displays a subset of in-app notifications based on defined rules, making it easier to surface important notifications.

Reviewer 2: ```suggestion
In the Inbox components, you can use Tabs to organize in-app notifications into separate sections, creating a cleaner, more organized experience for your subscribers. Each tab displays a subset of in-app notifications based on defined rules, making it easier to surface important notifications.
```

![Filtering tabs by tags and data attributes](/images/inbox/filter-tabs-by-tags-and-data-attributes.png)

## Group notifications into tabs

Each tab is defined using the `tabs` prop of the Inbox component. Every tab object includes the following properties:
- **`label`**: A string that defines the tab’s title, displayed as the visible label in the UI
- **`filter`**: An object that defines which notifications are included in a particular tab.

### Using tags

> Tags can be assigned to any workflow in the Novu workflow editor. These tags can then be used in the filter object to group notifications from one or more workflows under the same tab.

Reviewer: @Aviatorscode2 

we should add workflow tags section link here, so user can read on how to create tags on the workflow

> For example, assign a 'promotions' tag to marketing workflows or 'security' and 'alert' tags to security workflows to group their notifications under dedicated tabs. To show all notifications in a tab, pass an empty array to the tags property.

Reviewer: ```suggestion
For example, assign a **promotions** tag to marketing workflows or **security** and **alert** tags to security workflows to group their notifications under dedicated tabs. To show all the notifications in a tab, pass an empty array to the tags property.
```

```tsx
import { Inbox } from '@novu/react';

> function Novu() {

Reviewer: @Aviatorscode2 

Try to name the component as per its behaviour as `Novu` is generic

    />
  );
}

export default Novu;
```

### Using the data object

> Data object can be included in every in-app notification in the in-app editor editor. his object holds custom key-value pairs, which you can use to filter notifications into specific tabs.

Reviewer 2: ```suggestion
The data object can be included in every in-app notification in the in-app editor editor. This object holds custom key-value pairs, which you can use to filter notifications into specific tabs.
```

For example, use `{ priority: "high" }` to group all high-priority notifications under a dedicated "High Priority" tab.

```tsx
import { Inbox } from '@novu/react';

> function Novu() {

Reviewer: same here
